landoapi/app.py

```diff
@@ -55,6 +56,14 @@ def create_app(version_path):
     # Intialize the alembic extension
     alembic.init_app(app.app)
 
+    # Initialize rate limiter
+    # Most of the calls should come from other Conduit servers.
```

Comment:
i don't think it makes any sense to place rate limits on pingbacks from autoland-transplant.